.devcontainer/Dockerfile

```
@@ -0,0 +1,17 @@
+FROM mcr.microsoft.com/vscode/devcontainers/base:debian
+
+# Install additional OS packages.
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt update && apt install -y \
+  gcc \
+  gcc-10-multilib \
+  gcc-aarch64-linux-gnu \
+  gcc-x86-64-linux-gnu
+
+# Install Rust and Linux targets for the vscode user.
+USER vscode
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
+ARG PATH=/home/vscode/.cargo/bin/:$PATH
+RUN rustup target add \
```

```suggestion
RUN "${HOME}/.cargo/bin/rustup" target add \
```

Else I get an error about `rustup` not being found on my machine. I think something with updating `PATH` isn't working.

```
Step 7/7 : RUN rustup target add   aarch64-unknown-linux-gnu   x86_64-unknown-linux-gnu
 ---> Running in b84c45bf2a3d
/bin/sh: 1: rustup: not found
The command '/bin/sh -c rustup target add   aarch64-unknown-linux-gnu   x86_64-unknown-linux-gnu' returned a non-zero code: 127
```